docs/ml/deploy.md

```diff
@@ -50,6 +50,11 @@ For some models of the ML model service, like the [Triton ML model service](http
 
 ## Create an ML model service
 
+The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
+You can see all publicly available ML models as well as all ML models you've uploaded yourself in the [Viam Registry](https://app.viam.com/registry).
+
+Once you've decided on a model to use, copy the name of the ML model.
```

Comment:
I would remove this line. Here we advise that the user can copy the name of a chosen ML model, but all future references to this ML model selection explicitly specify the built-in only: `TFLite CPU`. If we do want to retain "you can pick any", following copy should likely also specify any rather than specific. Example for line 63:

> Select the `ML Model` type, then select the ML model you would like to use, such as the built-in `TFLite CPU` model.

Reply:
it's the ml model not the model name so "effdet0" for example...

Reply:
oh I see that doesn't make sense, agreed
```suggestion
```

Reply:
maybe